lib/transbank/sdk/patpass/patpass_by_webpay/errors/integration_type_error.rb

```diff
@@ -0,0 +1,15 @@
+module Transbank
+  module Patpass
+    module PatpassByWebpay
+      module Errors
+        class IntegrationTypeError < ::Transbank::Patpass::Errors::PatpassError
+          attr_reader :code
+          def initialize(message, code)
```

Comment:
This doesn't need a `code` (in the other errors the `code` is the HTTP response code)